longevity_test.py

```diff
@@ -44,8 +44,9 @@ def _run_all_stress_cmds(self, stress_queue, params):
 
             # Run all stress commands
             self.log.debug('stress cmd: {}'.format(stress_cmd))
-            stress_queue.append(self.run_stress_thread(**params))
-            time.sleep(2)
+            stress_queue.append(self.run_stress_thread(**params) if stress_cmd.startswith('cassandra-stress')
+                                else self.run_stress_thread_bench(stress_cmd=stress_cmd, stats_aggregate_cmds=False))
```

Comment:
improve readability: as a good practice, use ternary conditional operator when it can fit in one line